services/gateway/apis/http-version3/swagger/definitions/pos.json

```diff
@@ -107,74 +107,92 @@
             }
         }
     },
-    "unlock": {
+    "PoSUnlock": {
```

Comment:
what case are we following? it should be posUnlock?

Reply:
We are using pascal case to define swagger param definitions everywhere. Currently updated the ones concerning this PR but will gradually update the remaining.